deposit/src/test/java/edu/unc/lib/deposit/normalize/Simple2N3BagJobTest.java

```diff
@@ -109,9 +109,9 @@ public void depositContainerWithMODSTest() throws Exception {
 		Bag depositBag = model.getBag(job.getDepositPID().getURI());
 		Resource primaryResource = (Resource) depositBag.iterator().next();
 
-		assertEquals("Folder label was not set", primaryResource.getProperty(dprop(model, label)).getString(), name);
+		assertEquals("Folder label was not set", primaryResource.getProperty(CdrDeposit.label).getString(), name);
 
-		assertEquals("Content model was not set", primaryResource.getPropertyResourceValue(fprop(model, hasModel))
+		assertEquals("Content model was not set", primaryResource.getPropertyResourceValue(RDF.type)
```

Comment:
Content models should no longer be getting added (like CONTAINER), instead it should be using the rdf types from the Cdr helper class.